app/serializers/page_category_serializer.rb

```diff
@@ -16,6 +16,10 @@ def description
   end
 
   def type
-    'guide'
+    guide || object.layout.identifier
+  end
+
+  def guide
+    {'article' => 'guide'}[object.layout.identifier]
```

Comment:
What do you mean? 

'article' needs to be transformed into 'guide' to keep backwards compatibility with the old API, since that is what hippo called it, and all the other layouts translate as is. I am sure we can remove this silliness once we retire the old category manager.